test/wallet-test.js

```diff
@@ -2153,21 +2155,21 @@ describe('Wallet', function() {
       uTXCount++;
 
       // Check
-      let senderBal = await wallet.getBalance();
-      assert.strictEqual(senderBal.tx, 6);
-      assert.strictEqual(senderBal.coin, 4);
-      assert.strictEqual(senderBal.confirmed, fund - (cTXCount * fee));
-      assert.strictEqual(senderBal.unconfirmed, fund - (uTXCount * fee));
-      assert.strictEqual(senderBal.ulocked, secondHighest);
-      assert.strictEqual(senderBal.clocked, secondHighest);
-
-      let recipBal = await recip.getBalance();
-      assert.strictEqual(recipBal.tx, 0);
-      assert.strictEqual(recipBal.coin, 0);
-      assert.strictEqual(recipBal.confirmed, 0);
-      assert.strictEqual(recipBal.unconfirmed, 0);
-      assert.strictEqual(recipBal.ulocked, 0);
-      assert.strictEqual(recipBal.clocked, 0);
+      const senderBal1 = await wallet.getBalance();
```

Comment:
Good idea to make these very explicitly different variables

Reply:
👍